dockerfiles/musl-ci-linux/Dockerfile

```diff
@@ -1,130 +1,50 @@
-
-
-FROM ubuntu:latest
-
 ARG VCS_REF=master
 ARG BUILD_DATE=""
 ARG REGISTRY_PATH=paritytech
 
-# metadata
-LABEL io.parity.image.authors="devops-team@parity.io" \
-    io.parity.image.vendor="Parity Technologies" \
-    io.parity.image.title="${REGISTRY_PATH}/musl-ci-linux" \
-    io.parity.image.description=" \
-rust target x86_64-unknown-linux-musl, openssl, zlib, musl tools clang" \
-    io.parity.image.source="https://github.com/paritytech/scripts/blob/${VCS_REF}/\
-dockerfiles/musl-linux/Dockerfile" \
-    io.parity.image.documentation="https://github.com/paritytech/scripts/blob/${VCS_REF}/\
-dockerfiles/musl-ci-linux/README.md" \
-    io.parity.image.revision="${VCS_REF}" \
-    io.parity.image.created="${BUILD_DATE}"
-
-ARG TARGET=x86_64-linux-musl
-WORKDIR /builds
-
-
-# config for x86_64-unknown-linux-musl
-COPY utility/musl-ci-linux-config /root/.cargo/config
+FROM ${REGISTRY_PATH}/ci-linux:production
```

Comment:
are you sure you want to inherit from the image tested against substrate? It's bulky and does not get released every night.
To have base rust and ubuntu:20.04 I suggest [this one](https://github.com/paritytech/scripts/blob/master/dockerfiles/base-ci/Dockerfile)